packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md

```diff
@@ -142,21 +144,21 @@ A full list of implementations, both open-source and proprietary, is available a
 
 ## SCIM facade
 
-Sometimes it isn't feasible for your cloud-based application to natively support a SCIM API. An alternative option is to build and host your own SCIM facade middleware that translates between the Okta SCIM API connection and the cloud application's proprietary API. The Okta integration connection is then made to this SCIM facade.
+Sometimes it isn't feasible for your cloud-based application to natively support a SCIM API. An alternative is to build and host your own SCIM facade middleware. It can translate between the Okta SCIM API connection and the cloud application's proprietary API. The Okta integration connection is then made to this SCIM facade.
 
 ## Provision to on-premises applications
 
-This provisioning guide targets cloud-based apps. For on-premise apps, see [Provision on-premises provisioning](https://help.okta.com/okta_help.htm?id=ext_OPP_configure) for details about the Okta agent-based provisioning solution.
+This provisioning guide targets cloud-based apps. For on-premises apps, see [Provision on-premises provisioning](https://help.okta.com/okta_help.htm?id=ext_OPP_configure) for details about the Okta agent-based provisioning solution.
```

Comment:
I checked and the name of the page has changed, thankfully - because Provision on-premises provisioning is just nuts for a title haha
```suggestion
This provisioning guide targets cloud-based apps. For on-premises apps, see [Provision on-premises applications](https://help.okta.com/okta_help.htm?id=ext_OPP_configure) for details about the Okta agent-based provisioning solution.
```